source/agora/common/Serializer.d

```diff
@@ -380,6 +380,18 @@ public void serializePart (T) (scope const auto ref T record, scope SerializeDg
             entry.serializePart(dg);
     }
 
+    // Pointers are handled as arrays, only their size must be 0 or 1
```

Comment:
Recursive / Cyclic pointers are not supported, right? Hopefully we'll never use them though.

Reply:
Good point, I guess that's why they aren't usually supported... And I suppose inner pointers will mess things too :/

Reply:
I understand what situations are used through unittest.